src/timeout.c

```diff
@@ -1145,7 +1148,10 @@ lantern_message(struct obj* obj)
             pline("Batteries have not been invented yet.");
         break;
     case OBJ_FLOOR:
-        You_see("a lantern getting dim.");
+        if (Hallucination)
+            You("witness the force of entropy at work.");
```

Comment:
Entropy isn't really a force or work; would be more correct just to say that "you witness the entropy increase", though none of these are particularly lantern-related. Perhaps the player can muse that we all must die eventually due to entropy?

Reply:
That just makes it better as a hallucination message, doesn't it? Hallucination messages _shouldn't_ make too much sense.